src/app/game/components/game-detail/game-detail.component.ts

```diff
@@ -10,11 +11,22 @@ export class GameDetailComponent implements OnInit {
 
   @Input()
   game?: Game;
+  email?: string;
+  pwd?: string;
 
   constructor() {
   }
 
   ngOnInit(): void {
+    this.email = localStorage.getItem('email');
+    this.pwd = localStorage.getItem('pwd');
```

Comment:
```suggestion
    this.email = localStorage.getItem('email') as string;
    this.pwd = localStorage.getItem('pwd') as string;
```

Si tu regardes la doc de la méthode `getItem()` de Storage tu vois que le type de retour est `string` ou `null`. Dans ce cas là, tu dois forcer angular à interpréter ton résultat en string même si c'est null.

L'autre possibilité aurait été de déclarer ton interface User comme ceci :
```ts
export interface User {
  email: string | null;
  password: string | null;
}
```
mais j'aime pas des masses cette façon de faire.

![image](https://user-images.githubusercontent.com/4210719/213994509-76538282-617f-453f-9593-0c72abf4db52.png)